tests/mvp-demo/mvp-demo.go

```diff
@@ -454,8 +455,9 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 					GinkgoWriter.Printf("failed to get PipelineRun for a release '%s' in '%s' namespace: %+v\n", release.Name, managedNamespace, err)
 					return false
 				}
+				tests.ExpectPipelineRunNotToFail(pipelineRun)
 				return pipelineRun.IsDone() && pipelineRun.GetStatusCondition().GetCondition(apis.ConditionSucceeded).IsTrue()
-			}, releasePipelineTimeout, pipelineRunPollingInterval).Should(BeTrue())
+			}, releasePipelineTimeout, pipelineRunPollingInterval).Should(BeTrue(), fmt.Sprintf("failed to see pipelinerun %q in namespace %q with a label pointing to %q to complete successfully", release.Name, managedNamespace, release.Namespace))
```

Comment:
Shouldn't be `pipelineRun.Name`?

```suggestion
}, releasePipelineTimeout, pipelineRunPollingInterval).Should(BeTrue(), fmt.Sprintf("failed to see pipelinerun %q in namespace %q with a label pointing to %q to complete successfully", pipelineRun.Name, managedNamespace, release.Namespace))
```